app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/FaceDetectionPage.xaml.cs

```diff
@@ -79,11 +79,21 @@ public partial class FaceDetectionPage : Page, INotifyPropertyChanged
         /// </summary>
         private ObservableCollection<Face> _resultCollection = new ObservableCollection<Face>();
 
+        /// <summary>
+        /// Headpose results container
```

Comment:
[Nit] `Container of face detection results with faces rotated by head poses`

Reply:
Done.